R/readGENEActiv.R

```diff
@@ -31,6 +31,14 @@ readGENEActiv = function(filename, start = 0, end = 0, progress_bar = FALSE,
                      x = gsub(pattern = "Device Model:", replacement = "",
                         x = fh[grep(pattern = "Device Model", x = fh)[1]]))
   
+  if (starttime == "2010-09-16 09:10:00:000") {
```

Comment:
@vincentvanhees Does the start time definitely reset to this exact date/time? My reading of the email thread was that it gets reset to some time in 2010, not necessarily always this same date in 2010. In Kenn Konstabel's example, Start Time did reset to 2010-**09-16** 09:10:00:000, but the (presumably correct) Page Time was 2023-**09-16** 21:00:00:500, so same exact month and day. Which makes me think that maybe only the year gets rest to 2010, and month and day remain as they were? 

So maybe the test condition should be `if (startsWith(starttime, "2010")` ?

Also, I was wondering if maybe we should just always read the Page Time, not the Start Time? I was looking at some .bin files, and in all of them Page Time is 5 seconds (sometimes 4.5 seconds) after the Start Time. Maybe this is what explains the [magic 5 second difference](https://github.com/wadpac/GGIRread/pull/55#issuecomment-1905975698) you were seeing between timestamps generated by your library and those from the manufacturer's GENEAread library?

Reply:
Mike sent me a test file via private message which had 2010-09-16 09:10:00:000 as header start time, identical to the start time in Kenn's file. This made me conclude that the full timestamp is predictable.

However, you are right that it may be safer to avoid such odd condition and use the time from the first page instead.
Good to hear that this then also addresses the 5 second offset. I will now make one extra commit to use page time by default and to omit the `+ 5`.

Reply:
Oh, I see! I shouldn't have assumed that you were extrapolating from just one example.
But this looks good!